utils/converter/src/main/java/io/apicurio/registry/utils/converter/ExtJsonConverter.java

```diff
@@ -104,7 +105,7 @@ public SchemaAndValue toConnectData(String topic, byte[] value) {
 
         long globalId = ip.getGlobalId();
 
-        SchemaLookupResult<JsonNode> schemaLookupResult = getSchemaResolver().resolveSchemaByGlobalId(globalId);
+        SchemaLookupResult<JsonNode> schemaLookupResult = getSchemaResolver().resolveSchemaByArtifactReference(ArtifactReference.builder().globalId(globalId).build());
```

Comment:
Maybe add a `fromGlobalId()` method to `ArtifactReference`?

Reply:
good idea, btw take a look inside of `ArtifactReference` all the comments at the bottom about validating the values, wdyt about that?

Reply:
I saw that, yeah.  I would probably leave that out for now.

Reply:
OK to leave the comments though I think.

Reply:
I'm not normally in favor of leaving in commented out code, but in this case I think it makes sense to show what the thinking was.